### PR TITLE
Add "supported browsers" table to README

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
I just checked all these browsers/OS combinations to finally figure out (for realz) what platforms we currently support.  I think this is a much better overview than what we had before. Only thing I could not test is MacOS.

And I know that "Linux" is not a very precise definition of operating system, but checking for different linux distributions would be madness IMO. I think it's fine to assume that the browsers work the same regardless of Linux flavor.

The data for Edge assumes #241 is merged.

[Rendered README](https://github.com/elan-ev/opencast-studio/blob/bbc0e3b2d5c4a3ed1c6e27b4e8cd0717b4de6eec/README.md)

### CODE REVIEW

**Comment by LukasKalbertodt** [comment]
Oh and by the way: I also tried testing on iOS Chrome/Firefox/Edge, but all of those browsers fail when validating Opencast settings. I'm not sure if Apple disables third party cookies for those. I did not try to find the cause for this yet, as we all still assume those browsers on iOS are functionally equivalent to Safari anyway.

File: README.md [hunk 1]
```diff
@@ -81,8 +81,21 @@ If you prefer to run a local **development** server directly, you can use instea
...
+| OS         | Browser    | Capture Camera | Capture Screen | Record | Notes |
+| ---------- | ---------- | -------------- | -------------- | ------ | ----- |
+| Windows 10 | Chrome 77  | ✔              | ✔              | ✔      |
+| Windows 10 | Firefox 68 | ✔              | ✔              | ✔      |
+| Windows 10 | Edge 44    | (✔) [#217](https://github.com/elan-ev/opencast-studio/issues/217) | ✘ | ✘ | See [#242](https://github.com/elan-ev/opencast-studio/issues/242)
+| Linux      | Chrome 77  | ✔              | ✔              | ✔      |
+| Linux      | Firefox 68 | ✔              | ✔              | ✔      |
+| Android    | Firefox 68 | ✔              | ✘              | ✔      |
+| Android    | Chrome 78  | (✔) [#217](https://github.com/elan-ev/opencast-studio/issues/217) | ✘ | (✔) [#243](https://github.com/elan-ev/opencast-studio/issues/243) | Fairly unusable
+| iOS        | Safari     | (✔) [#217](https://github.com/elan-ev/opencast-studio/issues/217) | ✘ | (✔) [#84](https://github.com/elan-ev/opencast-studio/issues/84) | Video rotated by 180° in recording
```
Comment (LukasKalbertodt) [inline README.md]: Note to self: add "requires enabling experimental feature" to notes
